ansible_collections/arista/avd/roles/eos_designs/docs/tables/wan-settings.md

```diff
@@ -0,0 +1,161 @@
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;ike_policy_name</samp>](## "wan_ipsec_profiles.data_plane.ike_policy_name") | String |  |  |  | Name of the IKE policy. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;sa_policy_name</samp>](## "wan_ipsec_profiles.data_plane.sa_policy_name") | String |  |  |  | Name of the SA policy. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;profile_name</samp>](## "wan_ipsec_profiles.data_plane.profile_name") | String |  |  |  | Name of the IPSec profile. |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;shared_key</samp>](## "wan_ipsec_profiles.data_plane.shared_key") | String | Required |  |  | The IPSec shared key.<br>This variable is sensitive and SHOULD be configured using some vault mechanism. |
+    | [<samp>wan_mode</samp>](## "wan_mode") | String |  |  | Valid Values:<br>- <code>autovpn</code><br>- <code>sdwan</code> | Select if the WAN should be run using Pathfinder (`sdwan`) or Auto VPN only.<br>The default is `sdwan`<br>PREVIEW: This key is currently not supported |
+
+=== "YAML"
+
+    ```yaml
+    # PREVIEW: This key is currently not supported
+    # The AutoVPN RRs information for the WAN.
+
+    # When the AutoVPN RR is part of the same inventory as the WAN routers,
+    # only the name is required.
+    autovpn_rrs:
+
+        # Route-Reflector hostname.
+      - hostname: <str; required; unique>
+
+        # Route-Reflector router id.
+        router_id: <str>
+        carriers:
+
+            # Carrier name.
+          - name: <str>
+
+            # The public IP address for this carrier.
+            ip_address: <str>
+
+    # PREVIEW: This key is currently not supported
+    # List of carriers used for the WAN configuration.
+    carriers:
+
+        # Carrier name.
+      - name: <str; required; unique>
+
+        # Additional information about the carrier for documentation purposes.
+        description: <str>
+
+        # Flag to configure IPsec on the carrier (default is True).
+        ipsec: <bool>
+
+        # Indicates if the carrier has access to the Internet (`public`)
+        # or not (`private).
+        accessibility: <str; "public" | "private">
+
+        # A list of characteristics to assign to the carrier.
+        # TODO: Explain further how these are used (or removed before removing preview).
+        characteristics:
+          - <str; "backup" | "metered">
+
+        # TBC - edge or transit or both.
+        # Unclear if this should be at the carrier level or some other place.
+        circuit_type: <str; "edge" | "transit" | "both">
+
+    # PREVIEW: This key is currently not supported
+    # The Pathfinder(s) information for the WAN.
+
+    # When the pathfinder is part of the same inventory as the WAN routers,
+    # only the name is required.
+    sdwan_pathfinders:
```

Comment:
Why did we choose to omit the notion of a a pathfinder cluster and allow specifying what pathfinders a device peers with by specifying just the cluster?  All deployments will require atleast 2 pathfinder AFAIU.

Reply:
You can configure a WAN topo with only one pathfinder

Reply:
lets discuss this further on a call but Stun + AutoVPN works with one route-reflector.

Reply:
Logged in future work depending on testing once more of the solution is in place